docs/tutorials/tutorial-kafka.md

```diff
@@ -24,260 +24,267 @@ sidebar_label: "Load from Apache Kafka"
-
-Please follow the [query tutorial](../tutorials/tutorial-query.md) to run some example queries on the newly loaded data.
-
-## Cleanup
-
-To go through any of the other ingestion tutorials, you will need to shut down the cluster and reset the cluster state by removing the contents of the `var` directory in the Druid home, as the other tutorials will write to the same "wikipedia" datasource.
-
-You should additionally clear out any Kafka state. Do so by shutting down the Kafka broker with CTRL-C before stopping ZooKeeper and the Druid services, and then deleting the Kafka log directory at `/tmp/kafka-logs`:
-
-```bash
-rm -rf /tmp/kafka-logs
-```
+1. Click **Ingestion** in the console, then click the ellipses next to the refresh button and select **Submit JSON supervisor**.
 
+2. Paste this spec into the JSON window and click **Submit**.
+   ```json
+   {
+     "type": "kafka",
+     "spec": {
+       "ioConfig": {
+         "type": "kafka",
+         "consumerProperties": {
+           "bootstrap.servers": "localhost:9092"
+         },
+         "topic": "kttm",
+         "inputFormat": {
+           "type": "json"
+         },
+         "useEarliestOffset": true
+       },
+       "tuningConfig": {
+         "type": "kafka"
+       },
+       "dataSchema": {
+         "dataSource": "kttm-kafka-supervisor-console",
+         "timestampSpec": {
+           "column": "timestamp",
+           "format": "iso"
+         },
+         "dimensionsSpec": {
+           "dimensions": [
+             "session",
+             "number",
+             "client_ip",
+             "language",
+             "adblock_list",
+             "app_version",
+             "path",
+             "loaded_image",
+             "referrer",
+             "referrer_host",
+             "server_ip",
+             "screen",
+             "window",
+             {
+               "type": "long",
+               "name": "session_length"
+             },
+             "timezone",
+             "timezone_offset",
+             {
+               "type": "json",
+               "name": "event"
+             },
+             {
+               "type": "json",
+               "name": "agent"
+             },
+             {
+               "type": "json",
+               "name": "geo_ip"
+             }
+           ]
+         },
+         "granularitySpec": {
+           "queryGranularity": "none",
+           "rollup": false,
+           "segmentGranularity": "day"
+         }
+       }
+     }
+   }
+   ```
+
+
+   This starts the supervisor&mdash;the supervisor spawns tasks that start listening for incoming data.
+
+3. Click **Tasks** on the console home page to monitor the status of the job. This spec writes the data in the `kttm` topic to a datasource named `kttm-kafka-supervisor-console`.
+
+#### Use the API
+
+You can also use the Druid API to submit a supervisor spec:
+
+1. Run the following commands from your Druid root directory to download the sample spec:
```

Comment:
nit: this doesn't necessarily need to be done from the Druid root directory. The commands use curl.

Reply:
Updated.